index.html

```diff
@@ -8,18 +8,30 @@
   <link rel="stylesheet" href="./style.css">
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Poppins:wght@600&display=swap" rel="stylesheet">
 </head>
 <body>
-<header>
-  <div class="logo"><a href="#"><img src="./images/logo.png" alt="logo"></a></div>
-  <nav>
-    <div class="menu">
-      <a class="menu__topics" href="#"><span>Portfolio</span></a>
-      <a class="menu__topics" href="#"><span>About</span></a>
-      <a class="menu__topics" href="#"><span>Contact</span></a>
+  <div id="slide_menu">
+    <div><a href="#" class="cls">&times;</a></div>
+    <div>
+      <ul class="items">
+        <li><a class="menu__topics" href="#portfolio">Portfolio</a></li>
+        <li><a class="menu__topics" href="#about">About</a></li>
```

Comment:
Please Kindly fix this
This is you design: 
![Screenshot from 2022-05-12 20-22-29](https://user-images.githubusercontent.com/79506105/168125472-ba0bd7ee-c3ee-4c3e-854b-f14c053bd27b.png)

As you can see in your design the desktop Navigation is visible in the mobile display this isn't supposed to be visible in the mobile display 

2. Please fix the overflow when screen is reduced to mobile . This is why the Hamburger buttton isnt visible its because of the overflow on your mobile screen.
![Screenshot from 2022-05-12 20-22-51](https://user-images.githubusercontent.com/79506105/168126145-e46d7808-c697-4323-b62f-be942cf26897.png)